Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangleFEMForceField.inl

```diff
@@ -299,7 +308,13 @@ void TriangleFEMForceField<DataTypes>::accumulateForceSmall(VecCoord& f, const V
 
         // displacements
         Displacement Depl(type::NOINIT);
-        m_triangleUtils.computeDisplacementSmall(Depl, _rotatedInitialElements[elementIndex], deforme_b, deforme_c);
```

Comment:
To match TriangularFEMForceField code, I have updated initSmall to fill _rotatedInitialElements so it doesn't crash anymore if you restore this line.
In the end the results should be the same as the version you restored.

Reply:
Thanks @epernod